### PR TITLE
Fix read-more height error with img

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Vue.js has no way to know if img are loaded and in markdown description, imgs don't have any height from CSS so their sizes is based on the file size.

To solve this error, we redo the calculation after some time. We don't have any way to easily know ourself if imgs are loaded or not so it's a flat 500ms.

This seems to fix https://github.com/etalab/data.gouv.fr/issues/667

Close https://github.com/etalab/data.gouv.fr/issues/669

### CODE REVIEW

File: theme/js/components/utils/read-more.vue [hunk 1]
```diff
@@ -72,6 +67,16 @@ export default {
               ease: easing.anticipate,
             }).start(divStyler.set);
           }
+    },
+    updateReadMoreHeight() {
+      let contentHeight = Array.from(this.$refs.container.children)
+      .map(getHeight)
+      .reduce((total, height) => total + height, 0)
+      this.readMoreRequired = contentHeight > getHeight(this.$refs.container);
+      this.containerHeight = DEFAULT_HEIGHT;
```
Comment (maudetes) [inline theme/js/components/utils/read-more.vue:76]: Why do we need to set `containerHeight` again?

**Review by maudetes — COMMENTED** [review]
It seemed to have fixed both issues, even if difficult to reproduce!

File: theme/js/components/utils/read-more.vue (hunk 1 above)
Comment (nicolaskempf57) [reply theme/js/components/utils/read-more.vue:76]: Explained :)

**Review by maudetes — APPROVED** [review]
Seems all good :)

**Pull request merged**